server/controllers/entities/lib/properties/properties.js

```diff
@@ -0,0 +1,391 @@
+    subjectTypes: [ 'human' ],
+  },
+  // Anime News Network person ID
+  'wdt:P1982': {
+    subjectTypes: [ 'human' ],
+  },
+  // Anime News Network company ID
+  'wdt:P1983': {
+    subjectTypes: [ 'publisher' ],
+  },
+  // Anime News Network manga ID
+  'wdt:P1984': {
+    subjectTypes: [ 'work', 'serie' ],
+  },
+  // Twitter account
+  'wdt:P2002': {
+    subjectTypes: allLocallyEditedEntitiesTypes,
+  },
+  // Instagram username
+  'wdt:P2003': {
+    subjectTypes: allLocallyEditedEntitiesTypes,
+  },
+  // Facebook account
+  'wdt:P2013': {
+    subjectTypes: allLocallyEditedEntitiesTypes,
+  },
+  // YouTube channel ID
+  'wdt:P2397': {
+    subjectTypes: allLocallyEditedEntitiesTypes,
+  },
+  // BookBrainz author ID
+  'wdt:P2607': {
+    subjectTypes: [ 'human' ],
+  },
+  // number of volumes
+  'wdt:P2635': {
+    subjectTypes: [ 'edition' ],
+  },
+  // author of foreword
+  'wdt:P2679': {
+    subjectTypes: [ 'edition' ],
+  },
+  // author of afterword
+  'wdt:P2680': {
+    subjectTypes: [ 'edition' ],
+  },
+  // GoodReads author ID
+  'wdt:P2963': {
+    subjectTypes: [ 'human' ],
+  },
+  // GoodReads edition ID
+  'wdt:P2969': {
+    subjectTypes: [ 'edition' ],
+  },
+  // ISBN publisher prefix
+  'wdt:P3035': {
+    subjectTypes: [ 'publisher' ],
+  },
+  // Czech National Bibliography book ID
+  'wdt:P3184': {
+    subjectTypes: [ 'work', 'serie', 'edition' ],
+  },
+  // Babelio author ID
+  'wdt:P3630': {
+    subjectTypes: [ 'human' ],
+  },
+  // Babelio work ID
+  'wdt:P3631': {
+    subjectTypes: [ 'work', 'serie' ],
+  },
+  // Mastodon address
+  'wdt:P4033': {
+    subjectTypes: allLocallyEditedEntitiesTypes,
+  },
+  // MyAnimeList people ID
+  'wdt:P4084': {
+    subjectTypes: [ 'human' ],
+  },
+  // MyAnimeList manga ID
+  'wdt:P4087': {
+    subjectTypes: [ 'work', 'serie' ],
+  },
+  // Theses.fr person ID
+  'wdt:P4285': {
+    subjectTypes: [ 'human' ],
+  },
+  // British Library system number
+  'wdt:P5199': {
+    subjectTypes: [ 'edition' ],
+  },
+  // OCLC work ID
+  'wdt:P5331': {
+    subjectTypes: [ 'work', 'serie' ],
+  },
+  // BNB person ID
+  'wdt:P5361': {
+    subjectTypes: [ 'human' ],
+  },
+  // BD Gest' author ID
+  'wdt:P5491': {
+    subjectTypes: [ 'human' ],
+  },
+  // NooSFere book ID
+  'wdt:P5571': {
+    subjectTypes: [ 'edition' ],
+  },
+  // colorist
+  'wdt:P6338': {
+    subjectTypes: [ 'work', 'serie' ],
+  },
+  // LibraryThing author ID
+  'wdt:P7400': {
+    subjectTypes: [ 'human' ],
+  },
+  // BookBrainz work ID
+  'wdt:P7823': {
+    subjectTypes: [ 'work' ],
+  },
+  // BookBrainz publisher ID
+  'wdt:P8063': {
+    subjectTypes: [ 'publisher' ],
+  },
+  // Goodreads work ID
+  'wdt:P8383': {
+    subjectTypes: [ 'work', 'serie' ],
+  },
+}
+
+export const propertiesPerType = {}
+
+for (const type of allLocallyEditedEntitiesTypes) {
+  propertiesPerType[type] = []
+}
+
+for (const [ property, { subjectTypes } ] of Object.entries(properties)) {
```

Comment:
Do we expect a need for having the `subjectTypes` key ? Otherwise i would be for having a simpler structure:
```
{
  ...
  'wdt:P2635': [ 'edition' ],
  ...
}
```